common/test/data/scoreable/policies/assets.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Again, autogenerated course export.  If it wants an empty json dict, I'm not going to second guess.